extensions/amp-story/1.0/amp-story-quiz.js

```diff
@@ -252,13 +269,42 @@ export class AmpStoryQuiz extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Triggers the analytics event for quiz response
```

Comment:
This is very nitpicky but can you please add a period at the end of the comments please? Just trying to stick to the style guide as much as possible.

Reply:
Done, added punctuation to the JSDoc comments